aten/src/ATen/native/quantized/cpu/qconv.cpp

```diff
@@ -1644,6 +1614,91 @@ static at::Tensor _quantized_convolution_onednn(
     unary_scalars,
     unary_algorithm.has_value() ? unary_algorithm.value() : ""
   );
+
+#if IDEEP_PREREQ(3, 1, 0, 0)
+  // Use oneDNN's APIs instead of prepare/compute from ideep to reduce framework overhead.
+  // oneDNN version >= 3.1.0 is required.
+  using ideep::tensor;
+  static tensor empty_tensor;
+  bool has_postop_add = has_binary_post_op && binary_attr.value() == "add";
+  tensor src1 = has_postop_add ?
+      at::native::itensor_view_from_dense(accum.value().reshape({-1, accum.value().size(-1)})) :
```

Comment:
Why we reshape `accum` into two dimension?

Reply:
I have removed it. Thanks.